### PR TITLE
Updated RSelect styles

### PR DESCRIPTION
**Pull request by cesarlevel**
### What was a problem?

RSelect is not following Recomm's styles for select inputs, giving it a very inconsistent flavor.

### How this PR fixes the problem?

Updated a some classes to match Recomm's style

### Check lists

- [ ] Readme file updated with actual description and example
- [ ] Added all ARIA attributes required for component
- [ ] Added tests for both browser and SSR render and for all components properties
- [ ] Added story with all knobs and actions

### Additional Comments (if any)

Didn't update BEM style to SMACSS, will do in another ticket

### CODE REVIEW

**Comment by followyouhome** [comment]
Looks good

File: packages/recomponents/src/styles/theme.scss [hunk 1]
```diff
@@ -145,8 +145,8 @@ body, .article {
 
     // Elevation
     // ------------------------------------------------
-    --elevation-100: 0 0 0 1px var(--gray-color-light), 0 1px 2px 0 var(--gray-color-light), 0 4px 4px 0 var(--mono-color-100);
-    --elevation-200: 0 0 0 1px var(--gray-color-light), 0 7px 14px 0 var(--gray-color-light), 0 3px 6px 0 var(--gray-color-light);
+    --elevation-100: 0 0 0 1px rgba(74, 74, 75, 0.09), 0 1px 2px 0 rgba(74, 74, 75, 0.15), 0 4px 4px 0 rgba(74, 74, 75, 0.07);
+    --elevation-200: 0 0 0 1px rgba(74, 74, 75, 0.09), 0 7px 14px 0 rgba(74, 74, 75, 0.15), 0 3px 6px 0 rgba(74, 74, 75, 0.07);
```
Comment (followyouhome) [inline packages/recomponents/src/styles/theme.scss]: why not vars in this case?

Comment (cesarlevel) [reply packages/recomponents/src/styles/theme.scss]: I could create some `vars` from this but I want more time to think about which color I want. In the meantime, we could use this and update later on. Thanks for the review

**Pull request merged**